server/api/user/index.delete.ts

```diff
@@ -12,18 +12,25 @@ export default defineEventHandler(async (event) => {
     })
 
   // get request body
-  let { uid } = await readBody(event)
+  let { UIDs } = await readBody(event)
 
   // only admins can remove other users than their own
-  if (!hasAccess(user, ['admin']) || !uid) {
-    uid = user.uid
+  if (!hasAccess(user, ['admin']) || !UIDs) {
+    UIDs = user.uid
```

Comment:
I have the same inconsistency between `UID` and `uid` for event models/API and others in my branches. We may at some point have to discuss one of the two to use consistently, so it won't cause confusion for next years IT-group

Reply:
Hmm. You're right. I feel like for UIDs it makes most sence to use upper case. So I think we should change it to that.